source/dub/dub.d

```diff
@@ -1075,17 +1187,16 @@ class Dub {
 	private void updatePackageSearchPath()
 	{
 		if (m_overrideSearchPath.length) {
-			m_packageManager.disableDefaultSearchPaths = true;
+			m_packageManager.disableRepoSearchPaths = true;
 			m_packageManager.searchPath = [m_overrideSearchPath];
 		} else {
-			auto p = environment.get("DUBPATH");
-			Path[] paths;
-
 			version(Windows) enum pathsep = ";";
 			else enum pathsep = ":";
-			if (p.length) paths ~= p.split(pathsep).map!(p => Path(p))().array();
-			m_packageManager.disableDefaultSearchPaths = false;
-			m_packageManager.searchPath = paths;
+			m_packageManager.searchPath = environment.get("DUBPATH")
+				.splitter(pathsep)
+				.map!(p => makeAbsoluteRelativeToCwd(Path(p)))
```

Comment:
previously these paths were free-floating, seems better to make the absolute relative to what the cwd is *now* as opposed to when the path is actually used.